### PR TITLE
interoptest/testcoordinator: accept span proto from OC-Agent.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-ecosystem/opencensus-experiments/issues/84.

/cc @odeke-em

### CODE REVIEW

File: interoptest/src/testcoordinator/receiver/doc.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2018 Google LLC
```
Comment (odeke-em) [inline interoptest/src/testcoordinator/receiver/doc.go]: Wrong license headers, please use OpenCensus Authors.

File: interoptest/src/testcoordinator/receiver/sink.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// Copyright 2018 Google LLC
```
Comment (odeke-em) [inline interoptest/src/testcoordinator/receiver/sink.go]: Wrong license header please see https://github.com/census-ecosystem/opencensus-experiments/blob/680fbaa9af9d1ca864a863fe1f9fc4b2f35d6386/integration/go/grpc/client/interop_test.go#L1

File: interoptest/src/testcoordinator/receiver/receiver.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Google LLC
```
Comment (odeke-em) [inline interoptest/src/testcoordinator/receiver/receiver.go]: Wrong license header, please see https://github.com/census-ecosystem/opencensus-experiments/blob/680fbaa9af9d1ca864a863fe1f9fc4b2f35d6386/integration/go/grpc/client/interop_test.go#L1

File: interoptest/src/testcoordinator/receiver/doc.go (hunk 1 above)
Comment (songy23) [reply interoptest/src/testcoordinator/receiver/doc.go]: Good catch - not sure where I copied these headers. Fixed.

**Comment by songy23** [comment]
PTAL @rghetia.

Also @jparsana could you add @odeke-em to the collaborator list of this repo? Currently he doesn't have write access and we cannot request code review from him.

**Pull request merged**